src/box/errcode.h

```diff
@@ -115,286 +115,286 @@ struct errcode_record {
 
 #define ERROR_CODES(_) \
 	_(ER_UNKNOWN, 0,			"Unknown error") \
-	_(ER_ILLEGAL_PARAMS, 1,			"Illegal parameters, %s") \
-	_(ER_MEMORY_ISSUE, 2,			"Failed to allocate %u bytes in %s for %s") \
-	_(ER_TUPLE_FOUND, 3,			"Duplicate key exists in unique index \"%s\" in space \"%s\" with old tuple - %s and new tuple - %s") \
-	_(ER_TUPLE_NOT_FOUND, 4,		"Tuple doesn't exist in index '%s' in space '%s'") \
+	_(ER_ILLEGAL_PARAMS, 1,			"Illegal parameters", "details", STRING) \
+	_(ER_MEMORY_ISSUE, 2,			"Failed to allocate memory") \
```

Comment:
Need to specify how much and what for, as in the old error.

Reply:
Discussed f2f and decided to keep it this way as it only arises when converting from `OutOfMemory`. Yet message is changed to `"Out of memory"`.